Added error handling on TypeErrors

Makiyu-py opened the pull request:
#### Added some basic error handling when trying to do invalid operations.

- Made with <3 from Makiyu

Makiyu-py committed:
added more error handling

CodeWithSwastik reviewed (changes requested):
Just evaluate the portion of the tree only once and store it in a variable for referencing later, it'll be more efficient
Example:
For Plus Op
```python
tree1 = evaluate(tree[1])
tree2 = evaluate(tree[2])
try:
    return tree1 + tree2
except TypeError:
    return print(type_error.format(op="+", objtype1=str(type(tree1)).split("'")[1],
                                   objtype2=str(type(tree2)).split("'")[1]))
```

Makiyu-py committed:
fixed changes swas probably wanted lmao

Makiyu-py commented:
> Just evaluate the portion of the tree only once and store it in a variable for referencing later, it'll be more efficient
> Example:
> For Plus Op
> 
> ```python
> tree1 = evaluate(tree[1])
> tree2 = evaluate(tree[2])
> try:
>     return tree1 + tree2
> except TypeError:
>     return print(type_error.format(op="+", objtype1=str(type(tree1)).split("'")[1],
>                                    objtype2=str(type(tree2)).split("'")[1]))
> ```

fixed it!
